util/types/src/core/header_digest.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{core::views::HeaderView, packed, prelude::*, U256};
+use ckb_hash::new_blake2b;
+use ckb_merkle_mountain_range::{MerkleElem, Result as MMRResult};
+
+#[derive(Clone, Debug)]
+pub struct HeaderDigest {
+    data: packed::HeaderDigest,
+}
```

Comment:
I don't see why the wrapper is necessary. You can just `impl` anything for `packed::HeaderDigest`

Reply:
Updated, remove the wrapper.